internal/api/contract/overrides/schemas/Extern.yml

```diff
@@ -0,0 +1,12 @@
+description: Free-form Flux AST to prepend to query requests
+type: object
+properties:
+  type:
+    type: string
+    enum: [File]
+    default: File
+# NOTE: Purposefully type-less here because the boilerplate produced
```

Comment:
```suggestion
# NOTE: Intentionally type-less here because the boilerplate produced
```
Alternatively, I think "purposely" would work, but for some reason the spelling of that word just looks off to me :smile: